project/Dependencies.scala

```diff
@@ -40,7 +40,7 @@ object Dependencies {
   object Compile {
     // Compile
 
-    val camelCore     = "org.apache.camel"            % "camel-core"                   % "2.13.4" exclude("org.slf4j", "slf4j-api") // ApacheV2
+    val camelCore     = "org.apache.camel"            % "camel-core"                   % "2.17.1" exclude("org.slf4j", "slf4j-api") // ApacheV2
```

Comment:
If you are referring to binary compatibility between camel 2.13.4 and 2.17.1 with respect to the types required only by akka - it is time consuming, but possible to check.
Otherwise, if you refer to compatibility across all types and classes in camel core - I think it is difficult to assert unless camel dev know otherwise? This is my first PR, aside from common sense, what the hard rules around binary compatibility for Akka